main/persistent_storage/persistent_storage.c

```diff
@@ -51,30 +72,47 @@ esp_err_t persistent_storage_set_wifi(const char* ssid, const char* password) {
 	return ESP_OK;
 }
 
-esp_err_t persistent_storage_get_wifi(char* ssid, char* password) {
+esp_err_t persistent_storage_get_connection_data(
+	connection_data_t* connection_data
+) {
 	nvs_handle_t nvs;
 	PASS_ERROR(
 		nvs_open(NVS_NAMESPACE, NVS_READONLY, &nvs),
 		"Failed to open NVS storage"
 	);
 	LOG("NVS Open");
 
-	size_t max_length = WIFI_SSID_CHAR_BUFFER_LENGTH;
+	size_t max_length = WIFI_SSID_MAX_LENGTH;
```

Comment:
Waarom is deze variabele max, maaris de Sensor_id_length niet max?

Reply:
:shrug: 
It's a bit weirdly named I suppose, but no need to change it